rc_simulator/examples/annualSimulation.py

```diff
@@ -43,7 +43,7 @@
 
 
 # Initialise the Location with a weather file
-Zurich = Location(epwfile_path=os.path.join(
+Zurich = Location(path=os.path.join(
```

Comment:
I would still prefer this attribute to be called epfile_path as path can be confused with other potential attributes